backend/routes.js

```diff
@@ -110,3 +111,194 @@ module.exports = function routes(app, logger) {
     });
   });
 };
+
+
+//POST -> /auction -> create auction
+app.post('/auction', async (req,res)=>{
+  pool.getConnection(function (err, connection) {
+    if (err) {
+      // if there is an issue obtaining a connection, release the connection instance and log the error
+      logger.error("Problem obtaining MySQL connection", err);
+      res.status(400).send("Problem obtaining MySQL connection");
+    }else{
+      const id = req.body.id;
+      const current_bid = req.body.current_bid;
+      const list_user_id = req.body.list_user_id;
+      const bid_user_id = req.body.bid_user_id;
+      const product_id = req.body.product_id;
+      const show_user_bid = req.body.show_user_bid;
+      const is_finished = req.body.is_finished;
```

Comment:
Why would someone create an auction that was already finished? Think this should just be false